bugs.txt

```diff
@@ -0,0 +1,8 @@
+-card number allows non-numeric characters when at length 12.
```

Comment:
Yeah so for this file, remove it from your next commit (e.g. move it to your desktop).

1. Move bugs.txt to some location outside of git directory
2. `git status` see what files can be added to staging area
3. `git add .`
4. `git status` see what files have been put into the staging area
5. `git commit -m "your commit message"`
6. `git push`